lms/templates/ux/reference/teams-create.html

```diff
@@ -0,0 +1,200 @@
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/discussion/forum">
+                 Discussion
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/course_wiki">
+                 Wiki
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/progress">
+                 Progress
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/instructor">
+                 Instructor
+             </a>
+          </li>
+
+    </ol>
+  </div>
+</nav>
+</%block>
+
+<%block name="headextra">
+<%static:css group='style-course-vendor'/>
+<%static:css group='style-course'/>
+<%block name="nav_skip">${"#seq_content" if section_title else "#course-content"}</%block>
+</%block>
+
+<%block name="js_extra">
+
+  <script type="text/javascript">
+    $(function(){
+        $(".ui-accordion-header a, .ui-accordion-content .subtitle").each(function() {
+          var elemText = $(this).text().replace(/^\s+|\s+$/g,''); // Strip leading and trailing whitespace
+          var wordArray = elemText.split(" ");
+          var finalTitle = "";
+          if (wordArray.length > 0) {
+            for (i=0;i<=wordArray.length-1;i++) {
+              finalTitle += wordArray[i];
+              if (i == (wordArray.length-2)) {
+                finalTitle += "&nbsp;";
+              } else if (i == (wordArray.length-1)) {
+                // Do nothing
+              } else {
+                finalTitle += " ";
+              }
+            }
+          }
+          $(this).html(finalTitle);
+        });
+      });
+  </script>
+
+</%block>
+
+<div class="container">
+  <div class="course-wrapper">
+    <section class="course-content" id="course-content">
+
+    <header class="page-header">
+      <div class="header-main">
+        <nav class="breadcrumbs">
+          <a class="nav-item" href="">Solar Energy</a> <span class="icon fa-angle-right" aria-hidden="true"></span>
+        </nav>
+        <h2 class="page-title">Create a New Team</h2>
+        <p class="page-description">If you cannot find an existing team to join or would like to team up with a group of friends, create a new team.</p>
+      </div>
+    </header>
+
+    <div class="page-content">
+
+      <form class="create-team">
+      <fieldset class="required-wrapper">
+          <legend>Required Information</legend>
+
+          <div class="form-field required">
+            <label for="team-name" class="label">Team Name (Required)</label>
+            <input id="team-name" aria-required="true" class="form-input" />
+            <span class="tip tip-input">The name that will identify your team</span>
+          </div>
+
+          <div class="form-field required has-errors">
+            <label for="team-description" class="label">Team Description (Required)</label>
```

Comment:
Our pseudo-patterns in the past UI-wise was to note required fields by an asterisk, I think its definitely valuable to have the actual word "Required" in the label, but you may be covered by the ``required`` attr fairly well if you wanted to:

1. semantically include but visually hide the "(Required)" text
2. use an asterisk (with a note/description above all fields).